SingularityUI/app/controllers/TaskDetail.coffee

```diff
@@ -82,17 +82,36 @@ class TaskDetailController extends Controller
 
         app.showView @view
 
+        #  Refresh once again on page load to calculate initial CPU Usage value
+        @initialPageLoad = true
+        if @initialPageLoad
```

Comment:
@tpetr right, thanks for pointing out... will move to call second resource fetch after initial resourced fetched, where then I'll reset the state for future refreshes.